db-checkbook/build_scripts/01_dataloading.py

```diff
@@ -1,16 +1,33 @@
-from dcpy.connectors.s3 import client
+from dcpy.connectors import s3
+from functools import wraps
+from pathlib import Path
+import pandas as pd
+import geopandas as gpd
 
+base_bucket = 'edm-recipes'
+base_url = "https://edm-recipes.nyc3.cdn.digitaloceanspaces.com"
 
-def do_stuff() -> None:
-    print("doing stuff")
-    s3_client = client()
-    available_buckets = [
-        bucket["Name"] for bucket in s3_client.list_buckets()["Buckets"]
-    ]
-    print(f"This S3 client is a {type(s3_client)}")
-    print(f"This S3 client has access to buckets: {available_buckets}")
+
+
+# reading in the data: using S3 connectors
+
+# example: datasets/dcp_cpdb/2018_adopted_polygons/
+# dataset name: datasets/dcp_cpdb
+# version: 2018_adopted, 2019_adopted, 2020_adopted, 2021_adopted, 2022_adopted, 2023_executive
+# type geom: _polygons, _points
+
+def read_s3_edm_recipes_cpdb(dataset_name, version, type_geom, save_file_path):
+    digital_ocean_filepath = f'{dataset_name}/{version}{type_geom}/dcp_cpdb.shp'
+    s3_client = s3.client()
+    file = s3_client.download_file(base_bucket, digital_ocean_filepath, save_file_path)
+    return file
```

Comment:
This is maybe more specific than I'd hope, but didn't realize we had the complication of geometry separated by data, so this seems like roughly the best path for now.

Longer term would like just one file per cpdb version, but that would require merging of shapefiles. For now, use this, and I'll write up a data library ticket to get these in as single shapefiles and we can put that on hold for now